shippable.yml

```diff
@@ -0,0 +1,71 @@
+language: none
+
+runtime:
+    nodePool: shippable_shared_aarch64
+    # nodePool: shippable_shared_aarch32
+
+env:
+  global:
+  # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
+    - secure: bkTdATvev7sVFsP62xFV2ck215nXEtH7eWXdhzRRtbzeKquSkNhTGTCoa5FcLDvAVe36w+Sv59/3/oWNyMood8pIWjHLMC5CqqLdc4NRmyyaCKWys4CLhTTurIBPFSWUilxZW1KCKv/WHOe+zQDi2o9R9lf5/MizuwThHSQOIcqeTIn4wtPzbne5MeKSW+mRCsb+l4E/Q1oY2w/mTJ+izDWkxefstZ2t8RqOxH6H20wwNOOj/1WdeztdCOtCAl99r8Aj58odGyfUMAEyw89c5HglAEPurBQs21DZbHp10NmgSLyIbukplulRUm+cQ37loT/hFfTjPUCqLEC3lu6SPw==
+  matrix:
+    - CONFIG=linux_aarch64_python2.7
+    - CONFIG=linux_aarch64_python3.6
+    - CONFIG=linux_aarch64_python3.7
+
+
+build:
+  pre_ci_boot:
+    # It really doesn't like the centos image.....
+    # image_name: arm64v8/centos
+    # tag_name: 7
+
+    image_name: arm64v8/ubuntu
+    image_tag: 18.04
```

Comment:
Building this using ubuntu 18.04 would make the package built here incompatible with other centos7 and all the arm64v8 builds using numpy will have to be compiled with ubuntu 18.04 or newer.

Any reason not to use the `run_docker_build.sh` script?

Reply:
mostly just testing for now, to see if the non-native build is the cause of the compilation issue.  Appears that it isn't the case

Reply:
The exact container here is beside the point. I'm just hacking here on some old build scripts I have.

Shippable needs things like sudo, and very particular containers. The original error is still there, so I've ruled out the hypothesis that it was due to qemu

```
    numpy/core/src/multiarray/descriptor.c:2053:9: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
             Py_RETURN_FALSE;
             ^~~~~~~~~~~~~~~
    In file included from numpy/core/include/numpy/npy_math.h:548:0,
                     from numpy/core/src/npymath/npy_math_common.h:9,
                     from numpy/core/src/npymath/npy_math_complex.c.src:34:
    numpy/core/src/npymath/npy_math_internal.h.src: In function 'npy_cacoshf':
    numpy/core/src/npymath/npy_math_internal.h.src:482:12: internal compiler error: Segmentation fault
         return @kind@@c@(x, y);
                ^~~~~~~~~~~~~~~
    Please submit a full bug report,
    with preprocessed source if appropriate.
```